### PR TITLE
Sidebar-of-knative-revision-and-service| Knative Serverless

### PR DESCRIPTION
**Pull request by hemantsaini-7**
**Description:**

- Automation of side-bar-of-knative-revision-and-service.feature | Knative Serverless

**Jira Id:**
- Story: 
      - [ODC-7208](https://issues.redhat.com/browse/ODC-7208)
    

**Pre-conditions/setup:**

**Checks for approving Epic scenarios Automation PR:**

- [x] Execute the @to-do tagged gherkin scripts manually
- [x] Convert the @to-do gherkin scripts to cypress automation scripts
- [x] Once scripts are automated, replace tag @to-do with @epic-number
- [x] Execute the scripts in Remote cluster

**Refactoring criteria for approving Automation PR:**

- [ ] update the test cases count in [Automation status confluence Report](https://docs.jboss.org/display/ODC/Automation+Status+Report)

**Execution Commands:**
Example:
```
    export NO_HEADLESS=true && export CHROME_VERSION=$(/usr/bin/google-chrome-stable --version)
    BRIDGE_KUBEADMIN_PASSWORD=YH3jN-PRFT2-Q429c-5KQDr
    BRIDGE_BASE_ADDRESS=https://console-openshift-console.apps.dev-svc-4.12-042801.devcluster.openshift.com/
    export BRIDGE_KUBEADMIN_PASSWORD
    export BRIDGE_BASE_ADDRESS
    oc login -u kubeadmin -p $BRIDGE_KUBEADMIN_PASSWORD
    oc apply -f ./frontend/integration-tests/data/htpasswd-secret.yaml
    oc patch oauths cluster --patch "$(cat ./frontend/integration-tests/data/patch-htpasswd.yaml)" --type=merge
    ./test-cypress.sh -p knative
```

**Screen shots**:

image

**Browser conformance**:

- [x] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Edge

### CODE REVIEW

File: frontend/packages/knative-plugin/integration-tests/support/step-definitions/serverless/side-bar-of-knative-revision-and-service.ts [hunk 1]
```diff
@@ -73,28 +73,30 @@ When('user clicks on Actions dropdown in top right corner of side bar', () => {
 });
 
 Then(
-  'user able to see the options Edit Labels, Edit Annotations, Edit Revision, Delete Revision',
+  'user able to see the options Edit labels, Edit annotations, Edit Revision, Delete Revision',
```
Comment (sanketpathak) [inline frontend/packages/knative-plugin/integration-tests/support/step-definitions/serverless/side-bar-of-knative-revision-and-service.ts:76]: Use variables for the names

File: frontend/packages/knative-plugin/integration-tests/features/serverless/side-bar-of-knative-revision-and-service.feature [hunk 2]
```diff
@@ -42,33 +42,34 @@ Feature: side bar details
...
-        Scenario: Actions menu of knative service in side bar: KN-06-TC08
-            Given user has created knative service "nodejs-ex-git"
-             When user clicks on the knative service "nodejs-ex-git"
+        @regression
+        Scenario: Actions menu of knative service in side bar: KN-06-TC05
+            Given user has created knative service "nodejs"
+             When user clicks on the knative service "nodejs"
               And user clicks on Actions dropdown in top right corner of side bar
-             Then user able to see the options like Edit Application Grouping, Set Traffic Distribution, Edit Health Checks, Edit Labels, Edit Annotations, Edit Service, Delete Service
+             Then user able to see the options like Edit application grouping, Set traffic distribution, Edit Health Checks, Edit labels, Edit annotations, Edit Service, Delete Service, Edit resource limits, Edit nodejs
```
Comment (sanketpathak) [inline frontend/packages/knative-plugin/integration-tests/features/serverless/side-bar-of-knative-revision-and-service.feature:75]:
```suggestion
             Then user is able to see the options  "Edit application grouping", "Set traffic distribution", "Edit Health Checks", "Edit labels", "Edit annotations", "Edit Service", "Delete Service", "Edit resource limits", "Edit nodejs"
```

**Comment by sanketpathak** [comment]
/lgtm
Screenshot from 2022-12-09 16-55-27

**Comment by hemantsaini-7** [comment]
The pr only affects our E2E tests
/label qe-approved
/label docs-approved
/label px-approved

**Pull request merged**